test/e2e/network/kube_proxy.go

```diff
@@ -20,17 +20,20 @@ import (
 	"encoding/json"
 	"fmt"
 	"math"
+	"regexp"
 	"strconv"
 	"strings"
 	"time"
 
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
 
 	"k8s.io/kubernetes/test/e2e/framework"
+	"k8s.io/kubernetes/test/e2e/storage/utils"
```

Comment:
is this ok or the hostexec code should be moved to its own folder inside test/e2e/framework/hostexec?

Reply:
I wouldn't expect network e2e tests to use storage utils